test/services/sync/strategies/webuntis/WebUntisSyncer.test.js

```diff
@@ -0,0 +1,195 @@
+const { expect } = require('chai');
+const sleep = require('util').promisify(setTimeout);
+
+const app = require('../../../../../src/app');
+const testObjects = require('../../../helpers/testObjects')(app);
+const WebUntisSyncer = require('../../../../../src/services/sync/strategies/WebUntisSyncer');
+
+// These tests should use a mock once the webuntis sync actually does something!
+describe('webuntis Syncer', () => {
+	let server;
+	before((done) => {
+		server = app.listen(0, done);
+	});
+
+	after((done) => {
+		server.close(done);
```

Comment:
done is already called. Maybe the server.close is slower then the next step. But the save way is with await and async